traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -77,106 +77,107 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		// 1.2 routes are simply a Go replacement for the equivalent Perl route. They may or may not conform with the API guidelines (https://cwiki.apache.org/confluence/display/TC/API+Guidelines).
```

Comment:
can you change this to say 1.1 and 1.2 routes are simply....

Reply:
Done